test/transpile/module-resolution/project-with-tstl-library-has-exports-field/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "strict": true,
```

Comment:
I don't like we are being forced to add strict everywhere. What would happen if we don't do this, and which rules would we need to disable?

Reply:
One clean option is to just remove the strict everywhere and exempt the testing files from linting. Would you be okay with that? Otherwise you just have to exempt all the rules that require strict, I'm not sure which, it gets more complicated and harder to maintain.

Reply:
I don't want to remove the strict everywhere, I just don't want to see any strict changes in the diff for this pull request. Can you answer my question? What would happen if we don't add the stricts, and what rules would we need to disable to let eslint agree

Reply:
Yeah when I said "remote the strict everywhere" I meant only in my PR, not elsewhere in the project. I don't know how many rules conflict with strict, it would be a bunch of trial and error to figure out of commenting them out one by one. As I said before, maintaining a separate blacklist for conflicting rules is a pretty bad long term solution, so for now I am just using the legacy project behavior, which allows you to keep strict off.